click/termui.py

```diff
@@ -24,17 +24,18 @@ def hidden_prompt_func(prompt):
     return getpass.getpass(prompt)
 
 
-def _build_prompt(text, suffix, show_default=False, default=None):
+def _build_prompt(text, suffix, show_default=False, default=None, show_choices=False, choices=None):
     prompt = text
+    if choices is not None and show_choices:
+        prompt += ' (' + ", ".join(map(str, choices)) + ')'
     if default is not None and show_default:
         prompt = '%s [%s]' % (prompt, default)
     return prompt + suffix
 
 
-def prompt(text, default=None, hide_input=False,
-           confirmation_prompt=False, type=None,
-           value_proc=None, prompt_suffix=': ',
-           show_default=True, err=False):
+def prompt(text, default=None, hide_input=False, confirmation_prompt=False,
```

Comment:
Yes, I think it should be true by default.

Ideally it should also trunchate, but maybe for another time.